src/Resources/Subscriptions/index.ts

```diff
@@ -0,0 +1,30 @@
+import Client from '../../Client';
+import AbstractResource from '../AbstractResource';
+import Addons from './Addons';
+
+/**
+ * _Resources connected to subscriptions._
+ *
+ * - `$whirli.subscriptions.all();`
+ * - `$whirli.subscriptions.addons.all();`
```

Comment:
Seems we may be repeating stuff like this, as there's now the README as well.

Note than `$whirli` is a UWA thing, and not a client thing, so wouldn't make sense to write as such in this file